docs/usage.rst

```diff
@@ -435,7 +429,7 @@ Here are misc. notes about topics that are maybe not covered in enough detail in
 Item flags
 ~~~~~~~~~~
 
-`borg create -v` outputs a verbose list of all files, directories and other
+`borg create --changed` outputs a verbose list of all files, directories and other
 file system items it considered. For each item, it prefixes a single-letter
```

Comment:
i guess we could add something explicitely about --unchanged here, but the text is mostly correct to the exception of `U` files.